src/Task.php

```diff
@@ -73,7 +73,15 @@ public function run()
         } else {
             /**
              * Child
+             * We're detaching the child process from the parent so that if the main fpm restarts or is killed our process can continue. 
```

Comment:
can we rename the $pid and $sid variables to make it more explicit which is the parent and which is the child process